bin/atnodes

```diff
@@ -170,6 +173,7 @@ while (1) {
     while ($active_count < $concurrency) {
         last if !@hosts;
         my $host = shift @hosts;
+        my $login_with_password = (defined $password) && (!(defined $password_only_for_sudo));
```

Comment:
Maybe we could remove the `defined` part for `$password_only_for_sudo`? That variable is used as a boolean anyway :)